packages/nimble-components/src/wafer-map/tests/wafer-map.mdx

```diff
@@ -0,0 +1,23 @@
+import { DocsStory, Meta, Controls, Title } from '@storybook/blocks';
+import { NimbleWaferMap } from './wafer-map.react';
+import * as waferMapStories from './wafer-map.stories';
+
+<Meta of={waferMapStories} />
+<Title of={waferMapStories} />
+
+A wafer map is a component for visualizing data from the manufacture of semiconductor wafers. Each die on the wafer can show numerical information and be colored to indicate information about that die.
+
+<DocsStory of={waferMapStories.waferMap} expanded={false} />
```

Comment:
do we actually need to set expanded false on all these? They default to true? (quite possible but if so is a lame api, should be `collapse` or something if the default is `expanded`, boolean attributes should default to false)

Reply:
We should've been using the `Canvas` tag - swapped them all out - 👍